.circleci/config.yml

```diff
@@ -365,6 +365,28 @@ jobs:
       - store_artifacts:
           path: scripts/pro/implementation_coverage_full.csv
           destination: pro/implementation_coverage_full.csv
+      - persist_to_workspace:
```

Comment:
We persist the data we need as it's easier than to download via api. Thus, it might be a better option to add the upload in this step as well (considering that attaching the workspace takes some time)?

Reply:
I actually think it would even fit better in the reporting step, since we also send data to coveralls there. Tinybird is just another reporting backend. And we would save the 4 mins it takes to restore the compete workspace.